contracts/RibbonStakingRewards.sol

```diff
@@ -161,7 +161,7 @@ contract StakingRewards is
         uint256 reward = block.timestamp >= periodFinish.add(1 days) ? rewards[msg.sender] : 0;
         if (reward > 0) {
             rewards[msg.sender] = 0;
-            rewardsToken.safeTransfer(msg.sender, reward);
+            rewardsToken.safeTransfer(msg.sender, Math.min(reward, rewardsToken.balanceOf(address(this))));
```

Comment:
jhh set up was slightly off, so do not think we will have same issue. regardless as a safety net made it withdraw min between balance and reward.